addon/components/au-link.js

```diff
@@ -1,6 +1,30 @@
 import Component from '@glimmer/component';
+import { deprecate } from '@ember/debug';
 
 export default class AuLink extends Component {
+  get route() {
+    if (this.args.linkRoute) {
+      deprecate(
+        '@linkRoute is deprecated, use @route instead',
+        false,
+        {
+          id: '@appuniversum/ember-appuniversum.au-link.linkRoute-argument',
+          until: "0.7.0",
```

Comment:
Not sure about these version numbers but it's required by the deprecate util so I just provided something.